animation_nodes/data_structures/attributes/attribute.pxd

```diff
@@ -0,0 +1,28 @@
+cpdef enum AttributeType:
+    UV_MAP,
+    MATERIAL_INDEX,
+    VERTEX_COLOR,
+    CUSTOM
+
+cpdef enum AttributeDomain:
+    POINT,
+    EDGE,
+    POLYGON,
+    CORNER
+
+cpdef enum AttributeDataType:
+    INT,
+    FLOAT,
+    FLOAT2,
+    FLOAT_VECTOR,
+    FLOAT_COLOR,
+    BYTE_COLOR,
+    BOOLEAN
+
+cdef class Attribute:
+    cdef:
+        readonly str name
+        readonly AttributeType type
+        readonly AttributeDomain domain
+        readonly AttributeDataType dataType
+        readonly data
```

Comment:
Can this be `CList`?

Reply:
Omar, What is `CList`? Do you mean the lists that we have in AN?